### PR TITLE
Frontend views

### PR DESCRIPTION
**Pull request by nullcount**
Addresses #31 Adds app navigation and basic views for all the pages to build from

### CODE REVIEW

File: raincloud/__init__.py [hunk 1]
```diff
@@ -51,33 +51,33 @@ def chpasswd():
...
         # this route will store the license in a file somewhere for validating premium actions
-        return 
+        return
 
     # TODO: enable backups on the device
     @app.route('/backups/enable', methods=['POST'])
     def enableBackups():
         # call the ~/project_rainstorm/scripts/backups_enable.sh script
-        return 
-
```
Comment (WillHHippo) [inline raincloud/__init__.py:70]: We should add a linter to automatically catch these issues

File: raincloud/models/service.py [hunk 2]
```diff
@@ -33,27 +34,31 @@ def set_status(self, status):
...
                 return 'disabled'
         except docker.errors.NotFound:
             return 'disabled'
-    
+
+    def get_settings(self):
+        settingsFile = "{}/{}/{}".format(self.__services_folder(), self.name, "service.json")
+        if os.path.isfile(settingsFile):
+            return open(settingsFile, "r").read()
+        return {}
```
Comment (WillHHippo) [inline raincloud/models/service.py]: We should go ahead and return JSON here,

```suggestion
    service_file = "{0}/service.json".format(self.__service_folder())
        if os.path.isfile(settingsFile):
            with open(service_file) as f:
                return json.load(f)
        return {}
```

File: thunder/src/Components/ActivePage/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        setServices(data.data);
+      });
+  }, []);
+
+  return (
+    <div>
+      <List className={style.root}>
+        {services &&
+          services.map(service => {
+            const settings = JSON.parse(service.settings);
```
Comment (WillHHippo) [inline thunder/src/Components/ActivePage/index.js]: If we do the above in the server we don't have to parse the settings here.

**Review by WillHHippo — CHANGES_REQUESTED** [review]
Some small changes suggested. NACK?

File: raincloud/__init__.py (hunk 1 above)
Comment (nullcount) [reply raincloud/__init__.py:70]: Create issue to add prettier.io into our workflow

**Review by WillHHippo — DISMISSED** [review]
utACK

**Review by WillHHippo — APPROVED** [review]

**Pull request merged**